### PR TITLE
[C-1890] Improve artist-recommendations fetching/loading

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Improves mobile performance by only fetching artist-recommendations when a user is followed, rather than every time a profile is requested.

### CODE REVIEW

**Review by rickyrombo — APPROVED** [review]
How fast does the related artists load on Mobile? I think generally the job takes a bit if it needs refreshing, so I opted to prefetch

**Review by Kyle-Shanks — APPROVED** [review]

File: packages/web/src/common/store/profile/sagas.js [hunk 1]
```diff
@@ -324,20 +324,22 @@ function* fetchProfileAsync(action) {
...
+        )
+      }
     }
 
-    // Delay so the page can load before we fetch mutual followers
-    yield delay(2000)
-
     if (!isNativeMobile) {
+      // Delay so the page can load before we fetch mutual followers
+      yield delay(2000)
```
Comment (rickyrombo) [inline packages/web/src/common/store/profile/sagas.js:341]: ugh this is so jank 😂 - i wonder if there's a better way now?

**Comment by dylanjeffers** [comment]
> How fast does the related artists load on Mobile? I think generally the job takes a bit if it needs refreshing, so I opted to prefetch

From my testing it seemed quite quick!

File: packages/web/src/common/store/profile/sagas.js (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/common/store/profile/sagas.js:341]: So this is mobile web so will continue to ignore lmao!!

Comment (rickyrombo) [reply packages/web/src/common/store/profile/sagas.js:341]: it's not desktop web too??

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-1890-artist-recomendations

**Pull request merged**